projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/components/update-flag/update-flag.component.ts

```diff
@@ -151,4 +152,14 @@ export class UpdateFlagComponent implements OnInit {
       });
     }
   }
+
+  private getUpdateFlagNotEnteredErrorMessage(): UpdateFlagErrorMessage {
+    if (this.displayContextParameter === CaseFlagDisplayContextParameter.UPDATE) {
```

Comment:
@johnbenjamin-hmcts - as commented above, maybe could be rewritten as a `switch()` statement

Reply:
@Daniel-Lam - I have done the changes as you suggested.